### DISCUSSION TITLE
記事のハッシュタグを title 要素内容に入れないで欲しい

### DISCUSSION THREAD

**Discussion by debiru** [increments/qiita-discussions#591] [要望・アイデア・不具合]
例えば https://qiita.com/Qiita/items/c686397e4a0f4f11683d の記事の場合、記事タイトルは「Markdown記法 チートシート」ですが、title 要素内容は「Markdown記法 チートシート #Qiita - Qiita」になっています。

末尾の `- Qiita` は良いとしても、直前のハッシュタグ `#Qiita` の部分はユーザーの意図に反するタイトル要素内容になっていないでしょうか。このようなハッシュタグが勝手に付与されると記事を外部SNS等でシェアする際などに余計な作用をもたらします。

title 要素内容としてハッシュタグを出力するのを止めることを検討していただけないでしょうか。

**Comment by tanaka-asami415** [comment]
@debiru 
ご意見ありがとうございます🙏

意見が分かれるところでもあり、社内でも何度か議論をしたことがあるのですが
検索時にどういう記事かわかりやすくするために、現在の仕様とさせていただいております。
また、検索エンジン等で記事がより検索されやすくなるといったメリットもあります。

そのため、現時点ではハッシュタグの出力を止めることは予定しておりませんが
今回いただいたご意見は今後の参考にさせていただきます🙇
